src/turndown.js

```diff
@@ -237,6 +237,37 @@ turndownService.addRule("getpageshortcode", {
   }
 })
 
+/**
+ * Build quote element shortcodes for instructor insights sections
+ **/
+turndownService.addRule("quoteshortcode", {
+  filter: (node, options) => {
+    if (node.nodeName === "DIV" && node.getAttribute("class")) {
+      if (node.getAttribute("class").includes("pullquote")) {
+        return true
+      }
+    }
+    return false
+  },
+  replacement: (content, node, options) => {
+    try {
+      const children = Array.prototype.slice.call(node.childNodes)
+      const quoteP = children.find(
+        child =>
+          child.nodeName === "P" && child.getAttribute("class") === "quote"
+      )
+      const sigP = children.find(
+        child => child.nodeName === "P" && child.getAttribute("class") === "sig"
+      )
+      const quote = quoteP.textContent
+      const sig = sigP.textContent
+      return `{{< quote "${quote}" "${sig}" >}}`
```

Comment:
Do we need to do any escaping of quote characters which might be in `textContent`?

Reply:
That's probably a good idea, although I have been testing converting all courses and I don't believe there have been any issues.